src/containers/Welcome/Step4/Step4.tsx

```diff
@@ -215,7 +222,7 @@ const Step4 = (p: Wizard.StepProps) => {
               id="Step2-PolicyTerms"
               label={(
                 <Trans tOptions={{ lng: getCurrentCountryCheckbox(currentCountry) }} i18nKey="consent:agree">
-                  I have read, understood, and agree to the terms of the Virufy Privacy Policy.
+                  I have read, understood, and agree to the terms of the <LinkPurple to={'https://www.google.com/url?q=https://virufy.org/privacy_policy/&amp;sa=D&amp;source=editors&amp;ust=1677794208477099&amp;usg=AOvVaw29UQOOZ0EuNN2JukEkAXus'} target="_blank">Virufy Privacy Policy</LinkPurple>.
```

Comment:
Is the google url necessary?
Can we directly put the string into "to" prop?

Reply:
Do you mean that creating a new string variable above and put it into "to" prop? Instead of having the link in "to" prop?

Reply:
I mean put it directly, like this:
`to="https://www.google.com/url?q=https://virufy.org/privacy_policy/&amp;sa=D&amp;source=editors&amp;ust=1677794208477099&amp;usg=AOvVaw29UQOOZ0EuNN2JukEkAXus"`

And my other question was if this part of the link is necessary:
`https://www.google.com/url?q=`

Reply:
I made a change on this part, but it is not shown on this page. These are the changes I made. https://github.com/virufy/study/pull/123/commits/bbeffb1aa3a2978ee769344910cdb2d3b467a9f3